app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -248,6 +252,16 @@ def send_failed_email(user, enrollment)
     end
   end
 
+  def send_failed_fraud_email(user, enrollment)
+    user.confirmed_email_addresses.each do |email_address|
+      UserMailer.in_person_failed_fraud(
+        user,
+        email_address,
+        enrollment: enrollment,
+      ).deliver_now_or_later(**mail_delivery_params)
```

Comment:
```suggestion
      # rubocop:disable IdentityIdp/MailLaterLinter
      UserMailer.in_person_failed_fraud(
        user,
        email_address,
        enrollment: enrollment,
      ).deliver_later(**mail_delivery_params)
      # rubocop:enable IdentityIdp/MailLaterLinter
```
You should use `deliver_later` here (and in tests) instead of `deliver_now_or_later`. See the changes introduced in #7007

Reply:
thank you, i just updated it

Reply:
switched back to deliver_now_or_later for now until we figure out why deliver_later is not triggering an email when tested manually

Reply:
ok it's now deliver_now as we were able to confirm that it does work